frontend/packages/ceph-storage-plugin/src/models.ts

```diff
@@ -27,8 +27,8 @@ export const OCSServiceModel: K8sKind = {
 };
 
 export const CephBlockPoolModel: K8sKind = {
-  label: 'Ceph Block Pool',
-  labelPlural: 'Ceph Block Pools',
+  label: 'Block Pool',
+  labelPlural: 'Block Pools',
```

Comment:
Perhaps we should also change abbr ? 
CBP -> BP

Reply:
No need to change abbr, i just changed these labels for display, I don't think we are using abbr to display anywhere

Reply:
![image](https://user-images.githubusercontent.com/15073590/111323475-ec6ddb80-868f-11eb-95df-ce5cb7e03d7a.png)


only labels need to be displayed as BlockPool, other places like list and addr bar are fine( those are kind)

Reply:
okay